client/app/dedicated/nas/details/partition/nas-details-partition.html

```diff
@@ -58,124 +47,38 @@
-                                                })"
-                                                data-i18n-static="nas_partitions_action_update">
-                                        </button>
-                                    </li>
-                                    <li role="menuitem">
-                                        <a class="btn btn-link"
-                                           data-ui-sref="app.networks.nas.details.partition.access({ partitionName: partition.partitionName })"
-                                           data-i18n-static="nas_partitions_action_display_access">
-                                        </a>
-                                    </li>
-                                    <li role="menuitem"
-                                        data-ng-if="$ctrl.isNasha()">
-                                        <a class="btn btn-link"
-                                           data-ui-sref="app.networks.nas.details.partition.snapshot({ partitionName: partition.partitionName })"
-                                           data-i18n-static="nas_partitions_action_display_snapshot">
-                                        </a>
-                                    </li>
-                                    <li data-ng-if="$ctrl.data.nas.canCreatePartition">
-                                        <button type="button"
-                                                class="btn btn-link"
-                                                data-ng-click="$ctrl.data.nas.canCreatePartition && setAction('dedicated/nas/details/partition/delete/nas-partition-delete.html', partition)"
-                                                data-i18n-static="nas_partitions_action_delete">
-                                        </button>
-                                    </li>
-                                </ul>
-                            </div>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-            <div data-pagination-front
-                 data-items="$ctrl.table.partitionIds"
-                 data-paginated-items="$ctrl.table.partitionDetails"
-                 data-current-page="currentPage"
-                 data-nb-pages="nbPages"
-                 data-refresh="$ctrl.table.refresh"
-                 data-items-per-page="itemsPerPage"
-                 data-transform-item="$ctrl.transformItem(item)"
-                 data-on-transform-item-done="$ctrl.onTransformItemDone(items)"
-                 data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-                 data-item-per-page-placeholder="{{tr('pagination_display')}}">
-            </div>
+            <oui-datagrid data-rows-loader="$ctrl.loadDatagridPartitions($config)"
+                          data-row-loader="$ctrl.transformItem($row)">
+                <oui-column data-title="tr('nas_partitions_name')"
```

Comment:
Can't you use the `{{:: }}` notation just like below, with the `data-text` attributes ?